### PR TITLE
feat(tailwindcss): generate and bundle tailwindcss preset

### PR DESCRIPTION
**Pull request by borisdiakur**
# Description

- Generates and bundles Tailwind CSS preset.
- Fixes typo in design tokens file (fontFamiliy -> fontFamily)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/liquid/tailwindcss-integration.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+const liquidPreset = require('@emdgroup-liquid/liquid/dist/css/tailwind-preset.js')
+
+module.exports = {
+  presets: [liquidPreset],
+}
+```
+
+## Disabling Tailwind's default configuration
+
+Liquid's Tailwind preset extends Tailwind's [default configuration](https://unpkg.com/browse/tailwindcss@%5E2/stubs/defaultConfig.stub.js) by default. If you want Liquid's preset to **not** extend Tailwind's default configuration, include an empty `presets` key in the preset:
```
Comment (renet) [inline src/liquid/tailwindcss-integration.md:27]: This is also documented in Tailwinds docs. But if you want to explicitly mention it, I'd add that "not extending Tailwind" results in completely replacing the default config. This isn't clear by this paragraph.

**Review by renet — CHANGES_REQUESTED** [review]
After resolving these comments, PR can be merged.

File: src/liquid/tailwindcss-integration.md (hunk 1 above)
Comment (borisdiakur) [reply src/liquid/tailwindcss-integration.md:27]: I added an additional note.

**Review by renet — APPROVED** [review]

**Pull request merged**